packages/auth/migrations/20220621180000_create_access_token_revoked_column.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.alterTable('accessTokens', function (table) {
+    table.boolean('revoked').nullable()
+  })
+}
+
+exports.down = function (knex) {
+  return knex.schema.alterTable('accessTokens', function (table) {
+    table.dropColumn('revoked')
+  })
+}
```

Comment:
I think it's ok to just add the column to the access token table.

Reply:
Does it automatically create the column if I just add the field to the model class?